app/src/App.js

```diff
@@ -1,25 +1,22 @@
 import * as React from "react";
 
+import "react-datepicker/dist/react-datepicker.css";
+
 import { useAuth0 } from "@auth0/auth0-react";
 import dayjs from "dayjs";
+import Button from "react-bootstrap/Button";
 import Container from "react-bootstrap/Container";
+import Jumbotron from "react-bootstrap/Jumbotron";
 import Nav from "react-bootstrap/Nav";
 import NavDropdown from "react-bootstrap/NavDropdown";
 import Navbar from "react-bootstrap/Navbar";
 import Spinner from "react-bootstrap/Spinner";
-import {
-  BrowserRouter as Router,
-  Switch,
-  Route,
-  Link,
-  useParams,
-} from "react-router-dom";
+import { BrowserRouter as Router, Switch, Route, Link } from "react-router-dom";
```

Comment:
Nice job figuring out React Router.  I remember it wasn't easy for me when I first started learning it.